metrics/tests.py

```diff
@@ -100,9 +103,98 @@ def test_can_react_to_disjoint_content(self):
             'reaction': 1,
         }
 
-        serializer = ReactionSerializer()
+        serializer = HighlightedReactionSerializer()
 
         try:
             serializer.validate(data)
         except ValidationError:
-            self.fail("Reactions to diff. content should be valid")
+            self.fail("HighlightedReactions to diff. content should be valid")
+
+
+class ArticleReactionSerializerTest(TestCase):
+
+    def setUp(self):
+        p = Source(name="Test Paper")
+        p.save()
+        self.t = Topic()
+        self.t.save()
+        self.a = Article(headline="Test", url="http://test.com", source=p)
+        self.a.save()
+        self.a.topics.add(self.t)
+        self.u = User(username="test")
+        self.u.save()
+
+    def test_cant_react_to_article_twice(self):
```

Comment:
That would be done with a PATCH request, what's testing there is the result of two POST